bindings/Makefile

```diff
@@ -1,5 +1,7 @@
 SWAGGER_SPEC := ../proto/build/swagger/determined/api/v1/api.swagger.json
 CODEGEN_BIN := swagger-codegen-cli-2.4.14.jar
+py_bindings_dest := ../harness/determined/common/api/b.py
```

Comment:
can we have a more distinct name than `b.py`? `bindings.py`? `sdk_bindings.py`? `api_sdk.py`?

Reply:
the only benefit with b would be to make the name shorter when importing assuming it's usage scope is gonna be very high and everyone would know what it is. 

this remind me we should add a comment in the generated file to warn people from editing it.